plugin/src/main/java/io/snyk/eclipse/plugin/SnykStartup.java

```diff
@@ -53,16 +50,6 @@ public void earlyStartup() {
       @Override
       protected IStatus run(IProgressMonitor monitor) {
         try {
-          File cliFile = getCliFile();
-          if (!cliFile.exists()) {
```

Comment:
`disableRunAbortActions` should be called here, and the actions should be enabled on receiving/having a valid CLI Path.